<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceStatForm</class>
 <widget class="QWidget" name="InvoiceStatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发票统计</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblYear">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>年</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblMonth">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>月</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnScan">
       <property name="text">
        <string>重新扫描</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>收入</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="twIncome">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>成本</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="twCost">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>错误信息</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QListWidget" name="lwErrors"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>twIncome</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>InvoiceStatForm</receiver>
   <slot>doubleRecord(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twCost</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>InvoiceStatForm</receiver>
   <slot>doubleRecord(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lwErrors</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>InvoiceStatForm</receiver>
   <slot>doubleRecord(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doubleRecord(QModelIndex)</slot>
 </slots>
</ui>
